tests/lib/prepare.sh

```diff
@@ -154,6 +154,25 @@ setup_experimental_features() {
     fi
 }
 
+save_installed_core_snap() {
+    local target_dir="${1-}"
+
+    SNAP_MOUNT_DIR="$(os.paths snap-mount-dir)"
+    LIBEXEC_DIR="$(os.paths libexec-dir)"
+    core="$(readlink -f "$SNAP_MOUNT_DIR/core/current" || readlink -f "$SNAP_MOUNT_DIR/ubuntu-core/current")"
```

Comment:
is the ubuntu-core snap still a thing?
```suggestion
    core="$(readlink -f "$SNAP_MOUNT_DIR/core/current")"
```

Reply:
updated